assets/scripts/script.js

```diff
@@ -178,3 +170,17 @@ function textAreaSizesDependContent(element) {
   element.style.height = "1px";
   element.style.height = (26 + element.scrollHeight) + "px";
 }
+
+document.addEventListener("DOMContentLoaded", () => {
```

Comment:
We already have `DOMContentLoaded` listener, it calls `init`.
Why did you change it?